kuka_kr16_support/robots/kr16.urdf.xacro

```diff
@@ -4,22 +4,24 @@
   <xacro:arg name="rsi_ip" default="127.0.0.1"/>
   <xacro:arg name="rsi_port" default="49152"/>
   <xacro:arg name="socket_timeout" default="30"/>
+  <xacro:arg name="has_ft_sensor" default="false"/>
```

Comment:
The name of this argument can be discussed, in the UR urdf for example, `use_tcp_fts_sensor` is used and I think this is even a ros industrial convention.

Reply:
has_ft_sensor is a bit nicer imo